src/client/src/App/LiveRates/Tile/ExecutionResponse/Response.stories.tsx

```diff
@@ -33,63 +34,123 @@ export default {
       defaultNotional: 1000000,
     },
   },
-} as ComponentMeta<typeof StatelessExecutionResponse>
+} as Meta<typeof StatelessExecutionResponse>
 
 const TileContainer = styled.div`
   width: 360px;
   height: 180px;
   position: relative;
 `
 
-const Template: ComponentStory<typeof StatelessExecutionResponse> = (args) => (
-  <TileContainer>
-    <StatelessExecutionResponse {...args} />
-  </TileContainer>
-)
+export const Pending = {
```

Comment:
now, can we reuse any of this? .. that is meant to be the selling point of CSF 3